src/editor/Editor.js

```diff
@@ -1155,13 +1155,15 @@ define(function (require, exports, module) {
      *      the "ch" parameter.
      */
     Editor.prototype.setCursorPos = function (line, ch, center, expandTabs) {
+        var oldCursorPos = this.getCursorPos(true, "start");
```

Comment:
Looks like these the newly introduced events are not used in your PR? Do we still want to retain this change?